### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    choices = ["VIEWER", "REPORTER", "UPDATER", "DEVELOPER", "MANAGER"]
+    default = "REPORTER"
+
+[resources]
+
+    [resources.sources]
+
+        [resources.sources.main]
+        url = "https://sourceforge.net/projects/mantisbt/files/mantis-stable/2.25.7/mantisbt-2.25.7.tar.gz/download"
+        sha256 = "b47d7ef9764c50f7aed11ee07a663e841229dbd1ae7e056d94d976530c2df7fe"
```
Comment (Tagadda) [inline manifest.toml:57]:
```suggestion
        [resources.sources.main]
        url = "https://sourceforge.net/projects/mantisbt/files/mantis-stable/2.25.7/mantisbt-2.25.7.tar.gz/download"
        sha256 = "b47d7ef9764c50f7aed11ee07a663e841229dbd1ae7e056d94d976530c2df7fe"
        format = "tar.gz"
        extract = true        
```

**Comment by ericgaspar** [comment]
!testme

**Pull request closed without merging**